### ISSUE TITLE
Slim Skeleton shows 500 error right after setup 

### ISSUE THREAD

**Issue by gopumon** [slimphp/Slim-Skeleton#119]
Right after `composer create-project slim/slim-skeleton myslimsite`, I ran `composer start` and went to `localhost:8080/users/ `which returned a json object as below.
```
{
    "statusCode": 404,
    "error": {
        "type": "RESOURCE_NOT_FOUND",
        "description": "Not found."
    }
}{
    "statusCode": 500,
    "error": {
        "type": "SERVER_ERROR",
        "description": "ERROR: ob_clean(): failed to delete buffer. No buffer to delete on line 28 in file \/Volumes\/WD Mac\/work\/test\/slim-project\/myslimsite\/src\/Application\/ResponseEmitter\/ResponseEmitter.php."
    }
}
```
Any help on this much appreciated.

**Comment by gopumon** [comment]
**Update:**

I just changed the _line 28_ of _ResponseEmitter.php_ to `if(ob_get_contents()) ob_clean();` which took off the 500 Server Error, but still have problem with route `/users/` _(Trailing Slash)_ but no 404 with `route /users`.

I installed Trailing Slash middleware as mentioned in the documentation by
```
composer require middlewares/trailing-slash
```
and changing `/app/middleware.php` to
```
<?php
declare(strict_types=1);

use App\Application\Middleware\SessionMiddleware;
use Middlewares\TrailingSlash;
use Slim\App;

return function (App $app) {
    $app->add(new TrailingSlash(true));
    $app->add(SessionMiddleware::class);
};
```
Still no luck with trailing slashes, shows 404 for routes with /.

**Comment by erwinsetiawan** [comment]
Same here I got 404 after installation. Any help?

**Comment by erwinsetiawan** [comment]
Oh thats my Nginx config @gopumon check your webserver config. It might be wrong with your web server configuration

**Comment by timogoosen** [comment]
Come on to the Slim slack and maybe someone can help you:
slimphp.slack.com

**Comment by gopumon** [comment]
> Come on to the Slim slack and maybe someone can help you:
> slimphp.slack.com

Thanks @timogoosen , but I don't have an invitation on that Slack workspace.

The issue I face is the middleware TrailingSlash is not modifying the routes. The uri without trailing slashes works. The middleware have no effect. Initiates middleware on routes without trailing slashes.

I got rid of the 500 error with a simple if statement as mentioned above, don't know why that `ob_clean()` is called there and my fix is good or not.

**Comment by akrabat** [comment]
This question is better asked on our [Slack](https://www.slimframework.com/#community) and/or our [discourse forum](http://discourse.slimframework.com/).

**Comment by timogoosen** [comment]
@gopumon you can join the slack here:
https://join.slack.com/t/slimphp/shared_invite/enQtNjMxMjExMzYxMTU0LWVmNWQ1MWMxODA0ZjI2YTdkYzIyYjcxYTA3ZGQ2ZDNhZjdmMGVlODhlYWQ1ZmNhNTdiOWNlNmEzNGRiMjA1Yzc

**Comment by l0gicgate** [comment]
> I just changed the line 28 of ResponseEmitter.php to if(ob_get_contents()) ob_clean(); which took off the 500 Server Error

This has been fixed via https://github.com/slimphp/Slim-Skeleton/pull/117

I haven't tagged a new release yet. I will later this week.

**Comment by l0gicgate** [comment]
Just released `4.0.1`. I'm closing this as resolved. Please join our Slack group for further help.
